### PR TITLE
Add Desert Oasis 🏜

### PR DESCRIPTION
**Pull request by CaptainRex-501st**
### Description

I added a theme that is supposed to look like a desert oasis (beige/yellow for sand, blues for water, and greens for palm tree leaves). In the images the menu buttons are blue but they have been coded look like the error green. Also a big thanks to TeddiNotTeddy for helping me code my first ever theme!

unknown
unknown

### CODE REVIEW

**Comment by Miodec** [comment]
Text color needs to be as close to white or black as possible. It can have a little bit of color in it, but mainly monochrome.

**Comment by CaptainRex-501st** [comment]
I have changed the text color to be darker

image

**Comment by rizwanmustafa** [comment]
This seems like a fun theme. Can't wait to try it!

**Review by rizwanmustafa — CHANGES_REQUESTED** [review]
Rename the theme file to "desert_oasis.css"
Will maintain consistency and prevent any errors

File: static/themes/_list.json [hunk 1]
```diff
@@ -643,5 +643,10 @@
     "name": "moonlight",
     "bgColor" : "#1f2730",
     "textColor":"#c69f68"
-  }  
+  },
+  {
+     "name": "Desert Oasis",
```
[the remark this answers is not in the document]
Comment (Miodec) [reply static/themes/_list.json:648]: Also change the .css file name to all lowercase letters.

[the remark this answers is not in the document]
Comment (CaptainRex-501st) [reply static/themes/_list.json]: Ok, done! Anything else I need to change?

**Review by teddinotteddy — APPROVED** [review]
Looks good.

File: static/themes/_list.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (rizwanmustafa) [reply static/themes/_list.json]: Looks great.

[the remark this answers is not in the document]
Comment (rizwanmustafa) [reply static/themes/_list.json]: @Miodec. please approve.

**Pull request merged**